hack/lib/etcd.sh

```diff
@@ -102,7 +102,7 @@ kube::etcd::cleanup() {
 
 kube::etcd::install() {
   (
-    cd "${KUBE_ROOT}/third_party"
+    cd "${KUBE_ROOT}/third_party" || return
```

Comment:
won't `set -o errexit` accomplish the same thing for us?

Reply:
It will... and it's set from `init.sh`, which sources this. Good catch. I'll remove the patch.